platform/linux-generic/odp_init.c

```diff
@@ -147,6 +147,11 @@ static int read_configfile(void)
 	return 0;
 }
 
+void odp_init_param_init(odp_init_t *param)
```

Comment:
More than just consistency. Most applications today are initializing ODP with the call:

odp_init_global(&instance, NULL, NULL);

If they now want to specify a not_used feature, then they need to provide an odp_init_t as the 2nd argument. This API is used to initialize that to its default values in an implementation-independent manner.